### DISCUSSION TITLE
Is is possible to override the PR title to have the same title for every PR

### DISCUSSION THREAD

**Discussion by jdavis4452** [renovatebot/renovate#22897] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

2.6

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I am trying to get the PR title for every PR that renovate generates to be "Renovate Updates"

### What you tried so far.

In my default.json I currently have

  "commitMessageAction": "Renovate",
  "commitMessageTopic": "Updates",
  "commitMessageExtra": " "  

and my titles are coming out as

"Renovate all dependencies"

I also tried
  "commitMessageAction": "Renovate",
  "commitMessageTopic": "Updates",
  "commitMessageExtra": null

I tried this because the doc states that
commitMessage is made up of
{{{commitMessagePrefix}}} {{{commitMessageAction}}} {{{commitMessageTopic}}} {{{commitMessageExtra}}} {{{commitMessageSuffix}}}

The default for commitMessagePrefix and commitMessageSuffix is null so I didn't bother setting those.
I set values to commitMessageAction, commitMessageTopic, and commitMessageExtra to override the defaults

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
It might be possible, but it's strongly recommended against as Renovate uses PR title as part of its "primary key" for looking up PRs both open and closed

**Reply by jdavis4452** [reply]
So if it is possible then what I am doing wrong?
Where is the "all dependencies" coming from?

I understand it isn't recommended.
